style.css

```diff
@@ -0,0 +1,960 @@
+}
+
+.languages1 {
+  height: 24px;
+  left: 12px;
+  right: 78px;
+  top: calc(50% - 24px / 2);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #000;
+}
+
+.arrow {
+  padding-left: 130px;
+}
+
+.arrow1 {
+  padding-left: 188px;
+}
+
+.langcolor {
+  background-color: #f7f7f9;
+}
+
+.lang {
+  padding: 36px 36px 100px;
+}
+
+.js {
+  width: 303px;
+  height: 66.72px;
+  border-radius: 8px;
+  display: flex;
+  padding: 24px;
+  margin: 10px 0;
+  align-items: center;
+}
+
+hr {
+  width: 327px;
+}
+
+.js1 {
+  display: flex;
+  padding-left: 30px;
+}
+
+.nodots {
+  list-style-type: none;
+}
+
+.contact-box {
+  background-image: url(images/Contactme.png);
+  background-color: white;
+  background-repeat: no-repeat;
+}
+
+.cback {
+  color: #ebebff;
+  font-size: 20px;
+  font-weight: 100;
+  text-align: center;
+  padding: 24px;
+}
+
+.more {
+  width: 327px;
+  height: 102px;
+  left: 24px;
+  top: 4802px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  text-align: center;
+  color: #ebebff;
+  font-size: 20px;
+  line-height: 28px;
+  padding-bottom: 20px;
+}
+
+.bold1 {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  text-align: center;
+  color: #fff;
+  padding: 82px 24px 24px;
+}
+
+.input-text {
+  left: 16px;
+  right: 38px;
+  top: 15px;
+  bottom: 15px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #b3bac5;
+  margin-top: 2px;
+}
+
+input[type=text],
+select {
+  width: 100%;
+  padding: 18px 24px;
+  font-style: normal;
+  font-size: 17px;
+  font-weight: 400;
+  height: 18px;
+  left: 16px;
+  right: 38px;
+  top: calc(50% - 18px / 2);
+  background: #fff;
+  color: #172b4d;
+  display: flex;
+}
+
+input[type=submit] {
+  color: #6070ff;
+  padding: 16px 20px;
+  margin: 8px 0;
+  border: none;
+  border-radius: 8px;
+  cursor: pointer;
+}
+
+input[type=submit]:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+input[type=text],
+input[type=email],
+select,
+textarea {
+  width: 327px;
+  height: 48px;
+  display: flex;
+  padding: 20px;
+  margin: 8px 0;
+  border: 1px solid #ccc;
+  border-radius: 8px;
+  resize: vertical;
+}
+
+.input {
+  display: flex;
+  color: #6070ff;
+  font-weight: 600;
+  font-size: 18px;
+}
+
+textarea {
+  height: 170px;
+}
+
+/* ------------------------------------------------------------------------------------------------------------------- */
+
+/* -------------------------------------------D E S K T O P - L A Y O U T--------------------------------------------- */
+
+/* ------------------------------------------------------------------------------------------------------------------- */
+
+@media screen and (max-width: 768px) {
+  .nav1 {
+    display: none;
+  }
+
+  .card1 {
+    display: none;
+  }
+
+  .container {
+    display: none;
+  }
+}
```

Comment:
[OPTIONAL] Since you followed the industry-standard practice of mobile-first development, you don't need this media query here. Consider removing it, you can still keep the styling for these classes.